jenkins-integration/beaker/gatling/20_start_memory_watcher.rb

```diff
@@ -0,0 +1,27 @@
+test_name "Start memory watching script on SUT"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+logger_refresh = ENV['MEMORY_WATCHER_REFRESH'] || 300
+
+if not simulation_id
+  abort "GATLING_SIMULATION_ID environment variable required"
+end
+
+remote_logger_dir = "/root/#{simulation_id}_memory_usage"
```

Comment:
Since each simulation is getting its own gatling-puppet-load-test repo too, I might try to house all things related to that simulation in a single directory so it's easier to see all the things.

I'd probably consider creating an intermediate directory based on the simulation ID that we can put the gatling-puppet-load-test repo under, as well as this logging information.